transcom/mymove
# B 18163 ppm closeout rep should not adjust net shipment weight when customer is over their allowance

## [Agility ticket](tbd)

## Summary

Note: This branch adresses stories B-18163 & B-18300 as 18163 removes the ability to directly modify net weight, but would prevent tests passing to adress when weight is reduced. But 18300 provides the new means to modify net weight (indirectly)
Reimbursable Weight has been added to Review Weight Ticket to reflect maximum weight that can be reimbursed without modifying net weight. Therefore Edit net weight button is removed, instead when the full or empty weight fields are modified, the Net Weight is recalculated and the updated value displays.

##How to test
1. Login as a service counselor (under officelocal)
2. Select PPM Closeout find a move that has a PPM shipment that is over weight (above the allowable weight entitled)
3. Select 'Review Shipment Weights'
4. Click Review Documents under Weight moved by customer.
5a. The Allowable Weight should reflect the equivalent of NetWeight by default (if there is no allowable weight already set for this weight ticket. 
5b.Adjust the Full weight so that Full weight - Empty weight is <= allowable weight under PPM Net Weight. You can also do this by reducing full weight by "to reduce excess weight" under Net Weight. Note: The net weight value is updated after losing focus of either full or empty weight.
5b. There should be no direct way to adjust the net weight altering the values within full or empty weight should be the only way to modify the net weight.
6. when altering the weight for full or empty weight, the net weight should adjust (after the field is un-focused).
7.  after reducing the weight, the overweight notification should disappear.
8. after clicking confirm and continue navigate to Move details, the "This move has excess weight" banner should not be visible.
